### DISCUSSION TITLE
Prettier configuration to match GitBook's opinionated git sync markdown normalization

### DISCUSSION THREAD

**Discussion by petros** [GitbookIO/community#109] [Feature Requests]
### What's your feature/idea?

I would love to have a https://github.com/prettier/prettier configuration file officially maintained by GitBook. I would just add that to my git repository and it would format my markdown in a way that pleases the GitBook git sync algorithm.

### What's your desired outcome?

That would help me stick to the style GitBook prefers when synching to a git repository. It would reduce unnecessary commit changes when synching back from GitBook to a git repository. It helps a lot when most of the content comes from git repositories. I.e. we are not using the GitBook editor.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by biafffff** [comment]
Agree! See these 2 examples: 1st code from Gitbook, 2nd Markdown code shown in GitHub
![image](https://user-images.githubusercontent.com/26275332/221196041-bdc52062-d003-45ab-b67f-3d38a25f0d80.png)
![image](https://user-images.githubusercontent.com/26275332/221196080-2a396b99-3514-46fe-9eed-9d914a9283b1.png)

**Comment by rachellawson** [comment]
Also agree! This would save lots of confusion between those parties working in GitBook and those working in GitHub PRs. Being able to work in either environment is a key element in our choice of the platform

**Reply by rachellawson** [reply]
Though I will add that GitBook honoring a prettier file would be *even better*
